app/assets/stylesheets/partials/_table.scss

```
@@ -0,0 +1,10 @@
+table.govuk-table {
+  @include govuk-media-query($until: tablet) {
+  display: block;
+}
```

Can we just sort the spacing out for readability please?

```suggestion
  @include govuk-media-query($until: tablet) {
    display: block;
  }
```